pages/renew.tsx

```diff
@@ -574,13 +650,7 @@ export default function Renew() {
                 variant="solid"
                 onClick={handleSubmit}
                 loading={loading}
-                disabled={
-                  !applicantId ||
-                  !certified ||
-                  invalidPersonalAddress ||
-                  invalidContact ||
-                  invalidDoctor
-                }
+                disabled={!applicantId || !certified}
```

Comment:
Would it be possible to disable this button on any of the sections being invalid?

Reply:
I think we don't need that check because you can't proceed/navigate between forms until the current form you are on is valid. This means that you won't be able to reach this step to submit the renewal unless all of the other forms are already validated. As well, it would be a non-trivial change that would require effort to achieve

Reply:
Good point